script.js

```diff
@@ -0,0 +1,123 @@
+const theForm = document.querySelector(".formContainer")
+const confirmBtn = document.querySelector(".submitBtn")
+const continueBtn = document.querySelector(".Continue")
+const inputCardName = document.querySelector("#cardHolder")
+const inputCardNumber = document.querySelector("#cardNumber")
+const inputMonth = document.querySelector("#month")
+const inputYear = document.querySelector("#year")
+const inputCVC = document.querySelector("#cvc")
+const displayCardName = document.querySelector(".cardNameDisplay")
+const displayCardNumber = document.querySelector(".cardNumberDisplay")
+const displayMonth = document.querySelector(".displayMonth")
+const displayYear = document.querySelector(".displayYear")
+const displayCVC = document.querySelector(".displayCVC")
+const completeStateMessage = document.querySelector(".completeState")
+const incorrectName = document.querySelector(".incorrectName")
+const incorrectNumber = document.querySelector(".incorrectNumber")
+const incorrectDate = document.querySelector(".incorrectDate")
+const incorrectCVC = document.querySelector(".incorrectCVC")
```

Comment:
Look into using event delegation so that you can reduce the amount of variables you need to keep track of. You only need to reference the container elements.

https://javascript.info/event-delegation

Reply:
Also, take a look at this for an example on how to better organize your script:

https://github.com/1Marc/todomvc-vanillajs-2022/blob/main/js/app.js

What do you think about the structure and how do you think it might benefit you?